src/azureStorageExplorer/blobContainers/BlobContainerGroupTreeItem.ts

```diff
@@ -45,9 +45,9 @@ export class BlobContainerGroupTreeItem extends AzureParentTreeItem<IStorageRoot
         return !!this._continuationToken;
     }
 
-    async listContainers(continuationToken?: string, maxPageSize: number = 50): Promise<azureStorageBlob.ListContainersSegmentResponse> {
+    async listContainers(continuationToken?: string): Promise<azureStorageBlob.ListContainersSegmentResponse> {
         const blobServiceClient: azureStorageBlob.BlobServiceClient = this.root.createBlobServiceClient();
-        let response: AsyncIterableIterator<azureStorageBlob.ServiceListContainersSegmentResponse> = blobServiceClient.listContainers().byPage({ continuationToken, maxPageSize });
+        let response: AsyncIterableIterator<azureStorageBlob.ServiceListContainersSegmentResponse> = blobServiceClient.listContainers().byPage({ continuationToken, maxPageSize: maxPageSize });
```

Comment:
nit: Since the property name and constant name are the same we could do

```suggestion
        let response: AsyncIterableIterator<azureStorageBlob.ServiceListContainersSegmentResponse> = blobServiceClient.listContainers().byPage({ continuationToken, maxPageSize });
```